### ISSUE TITLE
STL files

### ISSUE THREAD

**Issue by rlogic4991** [sparkfun/Pokedex#3]
Hi! I was wondering how you dealt w the surface border errors in your older model when printing? Because I am not sure how to fix it and otherwise sketchup doesn't allow the part to be a solid. Also I was wondering if you had a measurement of surface area or volume of this model? I was trying to find it with sketchup so I could see how much it would cost me to print one for myself, but it wont list the volume of the case portion. 

Thanks

**Comment by rlogic4991** [comment]
Also how did you convert the sketchup files to stl? I am using solidworks for it because its the only other 3d modeling software I own (and i know it is far from optimal) but the model gets scaled by almost 10 times larger and I can't scale the model down in solidworks so what program did you use to create the stl file with dimensions intact?

**Comment by a5m0** [comment]
@rlogic4991 http://airwolf3d.com/wiki/how-to-convert-and-export-google-sketchup-skp-files-to-stl-for-3d-printing/

**Comment by jamesalmeida** [comment]
@rlogic4991 I used this to export .STL in Sketchup: https://extensions.sketchup.com/en/content/sketchup-stl

**Comment by rlogic4991** [comment]
Thanks!! I got it to work without a problem now

**Comment by DRA6N** [comment]
im getting errors when importing or opening in sketchup. would anyone be able to email me the stls?

**Comment by bboyho** [comment]
Hi,

I happened to come across this issue when checking out this repository. Sorry for the delay in reply. It looks like the original issue was resolved. The STL filess should be included in the folderes. I will be closing this issue out.

If you have errors opening the files up , try cloning the entire repository or downloading the zipped archive. Several people have reported that downloading the raw files individually yields corrupted models.
